src/actions/helpers.ts

```diff
@@ -1,6 +1,19 @@
-import { getFleekStorageService, StorageService } from "../storage";
+import {
+  getCloudinaryStorageService,
+  getFleekStorageService,
+  StorageService,
+} from "../storage";
 
-export const getStorage = (): StorageService => {
+import * as env from "env-var";
+
+export const getFleekStorage = (): StorageService => {
   const service: StorageService = getFleekStorageService();
   return service;
 };
+
+export const getCloudinaryStorage = (): StorageService => {
+  const service = getCloudinaryStorageService(
```

Comment:
are we unable to type this?

Reply:
We are, it just gets auto typed since `getCloudinaryStorageService` only ever returns a `StorageService`

Reply:
Typed it for consistency